.github/workflows/brand-plugin-test.yml

```diff
@@ -8,133 +22,64 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
-    name: Test Module in Plugin
+  setup:
+    name: Setup
     runs-on: ubuntu-latest
+    outputs:
+      branch: ${{ steps.extract_branch.outputs.branch }}
     steps:
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: "8.1"
-          coverage: none
-          tools: composer, cs2pr
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Checkout plugin
-        uses: actions/checkout@v4
-        with:
-          repository: bluehost/bluehost-wordpress-plugin
-          ref: 'main'
-
-      - name: Use Node.js 16.x
-        uses: actions/setup-node@v4
-        with:
-          node-version: 16.x
-          cache: 'npm'
 
       - name: Extract branch name
         shell: bash
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
```

Comment:
@circlecube  We should be able to get rid of this step, right?  It doesn't seem to be used anywhere now.

Reply:
Yea, it's already been removed. In order to get the workflow to run I had to have it on `main` so I added an initial version there and have since moved the real working parts to the reusable workflow. This makes the diff between main and this branch look a bit odd in the PR. The whole file is essentially being added though.

Reply:
Or at least I thought I did, I must have not had triggers set up properly or something, it is working in other repos to just add this workflow in a PR.